StokedSportsWebApi/wwwroot/index.js

```diff
@@ -1,18 +1,55 @@
-﻿
-async function sendAuthorizedRequestAsync(apiUrl, methodType, data) {
+﻿let response = await sendUnauthorizedRequestAsync("../api/Identity/Users", "POST", {})
+});
+function login(event) {
+    //store the username
+    //store password
+    //validate username and password
+    //send request to server to get a token 
+    //take user to secure page
+
+    let email = document.getElementById("txtEmail").value;
+    let password = document.getElementById("txtPassword").value;
```

Comment:
We should probably call your method and if it returns false we will return instead of continuing to login.